src/main.asm

```diff
@@ -1,26 +1,6 @@
-    struc sockaddr_in
-    .sin_family: resw 1
-    .sin_port: resw 1
-    .sin_addr: resd 1
-    .sin_zero: resq 1
-    endstruc
-
-    %define AF_INET 2
-    %define SOCK_STREAM 1
-
-    extern printf
-    extern dprintf
-    extern atoi
-    extern socket
-    extern htons
-    extern inet_aton
-    extern inet_ntoa
-    extern bind
-    extern close
-    extern listen
-    extern accept
-    extern dprintf
-    extern strlen
+    %include "c.hsm"
```

Comment:
On the stream I was struggling to come up with a good extension for "Assembly Header Files" so picked "hsm". But then I remembered that such files have "inc" extension. Even nasm manual refers to them http://www.nasm.us/doc/nasmdoc2.html#section-2.1.17

Let's rename `*.hsm -> *.inc`